.github/workflows/daily_tests.yml

```
@@ -53,6 +53,6 @@ jobs:
           username: ${{ secrets.MAIL_USERNAME }}
           password: ${{ secrets.MAIL_PASSWORD }}
           subject: NWB GUIDE Daily Test Failure
-          to: ${{ secrets.DAILY_FAILURE_EMAIL_LIST }}
+          to: cody.c.baker.phd@gmail.com
```

```suggestion
          to: cody.c.baker.phd@gmail.com, rly@lbl.gov
```
Can we add mine as well?

---

Of course!